docs/event-bus/docs/013-details-service-programming-model.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Service Programming Model
+type: Details
+---
+# Service Programming Model
+
+You can configure a microservice deployed in Kyma to receive Events from the Event Bus by creating a [subscription custom resource](https://github.com/kyma-project/kyma/blob/master/resources/cluster-essentials/templates/eventing-subscription.crd.yaml). Find various configuration options in the [Consume Events](https://github.com/kyma-project/kyma/blob/master/docs/event-bus/docs/011-details-event-flow-requirements.md#consume-events) section.
```

Comment:
please do not keep links to other docs in documentation
entry point for reader is kyma-project.io/docs and also docs ui in a cluster
for now linking in UI is not so easy but this doc will show up on the same page with the other doc you are trying to link to

Reply:
Thanks, please check now.